drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -233,8 +236,18 @@ def studio_view(self, context):
         for js_url in js_urls:
             fragment.add_javascript_url(self.runtime.local_resource_url(self, js_url))
 
+        # Do a bit of manipulation so we get the appearance of a list of zone options on
+        # items that still have just a single zone stored
+        data = copy.deepcopy(self.data)
```

Comment:
>Was that actually happening? I don't remember off the top of my head whether an XBlock write to a Scope.content field from a Studio student_view has any effect or not; there's a chance it does not.

So, it wouldn't just be opening the Studio view - it'd be if they opened the Studio view and then opened the editor for the block. Looking at the existing code, it looks like the database methods are completely abstracted - I can't tell if a method needs to be accessed in a particular way for an attribute change to be committed.

I'll see if I can do some further testing to verify the behavior; if it works like I think it does, then I'll add the comment @itsjeyd mentioned; if not, then I'll remove the `copy.deepcopy()` call.